package.json

```diff
@@ -34,24 +34,25 @@
   },
   "dependencies": {
     "@unly/utils-aws": "1.0.3",
-    "airtable": "0.7.2",
-    "aws-sdk": "2.534.0",
+    "airtable": "0.8.1",
+    "aws-sdk": "2.868.0",
     "dotenv": "8.0.0",
-    "epsagon": "1.37.0",
+    "epsagon": "1.106.1",
+    "inquirer": "7.0.0",
```

Comment:
This doesn't seem to be used elsewhere, is it a peer-dependency? 🤔

Reply:
I think I put that in when I saw this:
```bash
$ yarn install
...
warning "serverless > @serverless/components > inquirer-autocomplete-prompt@1.3.0" has unmet peer dependency "inquirer@^5.0.0 || ^6.0.0 || ^7.0.0".
```

I'm pretty new to the node.js/yarn world so I'm not sure what the "right" thing to do here would be. Possibly open an issue against `inquirer-autocomplete-prompt` to get them to fix their `package.json`?

Would you recommend removing this from the `package.json` in this PR? Not having it didn't seem to break anything, I just despise warnings. :)

Reply:
Rule of thumb of the NPM world, don't stress over warnings, only errors really matter. (99% of the time)

I'd remove it from package.json indeed. It's not our job to fix upstream misconfigurations, and it's probably harmless. Many people (packages authors) use version warnings but few do it correctly anyway.

So, can you remove it and confirm it doesn't affect the behaviour?

Reply:
Cool, I removed it and things still work fine. Let me know if you want any other changes on this PR.